scripts/elastic_search/files/txt_file.py

```diff
@@ -12,3 +12,9 @@ def get_data(self):
                 lines.append(line)
         f.close()    
         return lines
+
+    def get_GO_data(self):
```

Comment:
Indeed! That's a leftover function. I was trying to extract the GO data in batches and forgot to remove it when I changed my mind.